packages/react/src/app-layout/AppLayoutHeaderBrand.tsx

```diff
@@ -224,18 +217,27 @@ function AppLayoutHeaderBrandBadge({
 
 const DividingLine = ({
 	dividerPosition,
-}: Pick<AppLayoutHeaderBrandProps, 'dividerPosition'>) => (
+	singleLogo,
+}: Pick<AppLayoutHeaderBrandProps, 'dividerPosition'> & {
+	singleLogo?: boolean;
+}) => (
 	<Box
 		css={{
 			borderLeft: boxPalette.border,
 			borderLeftStyle: 'solid',
 			borderLeftColor: boxPalette.border,
+			margin: singleLogo ? '0 1rem' : undefined,
 			...packs.print.hidden,
 		}}
-		display={{
-			xs: 'none',
-			sm: dividerPosition === 'between' ? 'block' : undefined,
-			lg: 'block',
-		}}
+		display={
+			singleLogo
+				? { xs: 'none', md: 'block' }
+				: {
+						xs: 'none',
+						sm: dividerPosition === 'between' ? 'block' : undefined,
+						lg: 'block',
+				  }
+		}
+		height={singleLogo ? '3.5rem' : undefined}
```

Comment:
```suggestion
		height={singleLogo ? LOGO_HEIGHT : undefined}
```

Reply:
Awkwardly, `LOGO_HEIGHT` is `3.75rem`.

Reply:
Hmmm the design suggests they're both the same 
<img width="677" alt="image" src="https://github.com/agriculturegovau/agds-next/assets/853552/a3f34dce-7c0c-4dd8-ac3e-d6a0501658f4">

Reply:
Yeah, that's right. The existing code doesn't match design so [I'll update them all](https://github.com/agriculturegovau/agds-next/pull/1704#discussion_r1663469101).